scripts/generate_verify_psa_macros.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env python3
+
+"""Generate verify_psa_macros.c"""
+
+# Copyright The Mbed TLS Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import re
+
+OUTPUT_TEMPLATE = '''\
+/* Automatically generated by generate_verify_psa_macros.py. DO NOT EDIT. */
+
+#include <stdio.h>
+
+#include "psa/crypto.h"
+
+{header}
+
+#define hash_alg    PSA_ALG_SHA_256
+#define mac_alg     PSA_ALG_CMAC
```

Comment:
Missing: `aead_alg` (for `PSA_ALG_AEAD_WITH_DEFAULT_LENGTH_TAG` and `PSA_ALG_AEAD_WITH_SHORTENED_TAG`).